pkg/mgrconfig/load.go

```diff
@@ -94,12 +94,49 @@ func defaultValues() *Config {
 		Procs:          6,
 		PreserveCorpus: true,
 		Experimental: Experimental{
-			RemoteCover: true,
-			CoverEdges:  true,
+			RemoteCover:      true,
+			CoverEdges:       true,
+			DescriptionsMode: DescriptionsMode(ManualDescriptions).String(),
 		},
 	}
 }
 
+type DescriptionsMode int
+
+const (
+	ManualDescriptions = iota
+	AutoDescriptions
+	AnyDescriptions
+	Invalid
```

Comment:
Make Invalid first (0) value.
Then you can remove getDescriptionsMode function and just use `strToEnum[cfg.Experimental.DescriptionsMode]`.
May also help to catch bugs with uninitialized fields of this type (they will get Invalid value, rather than implicitly get ManualDescriptions value).

Reply:
Done!